cpp/ps0003/main.cpp

```diff
@@ -1,21 +1,23 @@
-
 #include <iostream>
 using namespace std;
 
 int main()
- {
-   int number;
-cout<<"please enter a number "<<endl;
-
-    cin >> number;
-    if (number % 2 == 0)
 {
-        cout << "even" << endl;
-    }
-else
-{
-        cout << "odd" <<endl;
-    }
-
-    return 0;
+   // prompting user for number 2/2
+   int number;
+   cout<<"please enter a number "<<endl;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number